lib/badge-data.js

```diff
@@ -58,7 +58,7 @@ function setBadgeColor(badgeData, color) {
 }
 
 function makeLabel(defaultLabel, overrides) {
-  return overrides.label || defaultLabel;
+  return overrides.label !== undefined ? overrides.label || '' : defaultLabel;
```

Comment:
This is pretty difficult to understand. Is it equivalent to this?
```js
return overrides.label !== undefined ? overrides.label : defaultLabel;
```

Probably it would be slightly more clear without the negation:
```js
return overrides.label === undefined ? defaultLabel : overrides.label;
```

Reply:
Did it this way _(`overrides.label || ''`)_ so can set `?label=false` to remove the label instead of only supporting `?label=`.

Can update it to use:
```javascript
return overrides.label === undefined ? defaultLabel : overrides.label;
```
If you prefer that though.

Will update to use `===` either way though.

Reply:
On the static badge it seems to return the string `'false'`
but on other services badges it returns Boolean `false` (also the same thing with numbers & `null`)
so `null`, `0`, `false` would all return an empty string. with the above logic.

Probably have to update the `makeLabel()` function to be:
```javascript
return '' + overrides.label === undefined ? defaultLabel : overrides.label;
```
if we want to always explicitly return a string.

Reply:
This has been sitting here a while!

What do you think about making `?label=` (empty string) the way to remove the label?

Reply:
I like that idea myself, seems intuitive

Reply:
:+1: 
Updated to only use `?label=`